plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/model/meta/GenericMetaModel.java

```diff
@@ -1061,4 +1061,7 @@ public boolean supportsViews(@NotNull GenericDataSource dataSource) {
             ((JDBCDataSourceInfo) dataSourceInfo).supportsViews();
     }
 
+	protected String getDefaultTypeName() {
+		return "N/A";  //$NON-NLS-1$
+	}
```

Comment:
Please use spaces and specify the `@NotNull` annotation for the return type.
```suggestion
    @NotNull
    protected String getDefaultTypeName() {
        return "N/A";  //$NON-NLS-1$
    }
```

Reply:
Thanks for your review, and sorry for my dirty code...